compiler/AST/foralls.cpp

```diff
@@ -178,6 +185,12 @@ void addForallIntent(CallExpr* call, ShadowVarSymbol* svar) {
   call->insertAtTail(svar->defPoint);
 }
 
+void checkTypeParamOuterVar(SymExpr* outerSE) {
```

Comment:
Since this produces an error specifically about forall/task intents, is there a way to make the function name reflect that? E.g. `checkTypeParamTaskIntent`